### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tiserett**
- [DEMO LINK](https://tiserett.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
+    transition: transform 0.3s, color 0.3s;
+
```
Comment (roma-shafihullin) [inline src/styles/_catalog.scss:18]:
```suggestion
    transition: transform 0.3s, color 0.3s;

```
by the way, you can set 'cursor: pointer;' in this element 😉

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
+    transition: transform 0.3s, color 0.3s;
```
Comment (roma-shafihullin) [inline src/styles/_catalog.scss:17]: by the way, you can set 'cursor: pointer;' in this element 😉

**Comment by roma-shafihullin** [comment]
Well done 👍

**Review by antutu93 — APPROVED** [review]

**Review by AlinaMatuschak — APPROVED** [review]
🚀

**Pull request closed without merging**